swift/Workflow/Tooling/Templates/install-xcode-templates.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
+
+# Configuration
+XCODE_TEMPLATE_DIR=$HOME'/Library/Developer/Xcode/Templates/File Templates/Workflow'
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# Copy workflow file templates into the local workflow template directory
+xcodeTemplate () {
+  echo "Copying workflow Xcode file templates..."
+
+  if [ -d "$XCODE_TEMPLATE_DIR" ]; then
+    rm -R "$XCODE_TEMPLATE_DIR"
```

Comment:
Should we be at all worried about people with preexisting `~/Library/Developer/Xcode/Templates/File Templates/Workflow` contents?

Reply:
Hmm. I've generally liked this, as over time when we've changed the templates it cleans up, and the name of the pod of `Workflow` is unique.

If you feel strongly, I can remove this though.

Reply:
I think the functionality is good. But I worry that `Workflow` is too generic to assume we own it. Maybe a more-namespaced directory would work?

Reply:
Yeah, I think you're right that it's too generic to make assumptions.

Updated and removed the rm -R